A32NX/html_ui/Pages/VCockpit/Instruments/A320_Neo/Map/Svg/A32NX_SvgConstraintElement.js

```diff
@@ -0,0 +1,61 @@
+class A32NXSvgConstraintElement extends SvgConstraintElement {
+    createDraw(map) {
```

Comment:
I've tried to override this function to avoid duplicate code but in the end, found out that the code was almost the same.